### PR TITLE
Convert validates to supports

### PR DESCRIPTION
**Pull request by kbrock**
part of https://github.com/ManageIQ/manageiq/issues/21179

converting AvaliableMixin to SupportsFeatureMixin

### CODE REVIEW

**Comment by kbrock** [comment]
WIP: using newer supports feature for string return.
need to update if that is not going in soon.

Also double check the localization

**Comment by kbrock** [comment]
un-WIP: just going with current valid syntax. can change later

File: app/models/manageiq/providers/openstack/infra_manager.rb [hunk 1]
```diff
@@ -27,6 +27,7 @@ class ManageIQ::Providers::Openstack::InfraManager < ManageIQ::Providers::InfraM
   supports :events do
     unsupported_reason_add(:events, _("Events are not supported")) unless capabilities["events"]
   end
+  not_supported :shutdown
```
Comment (agrare) [inline app/models/manageiq/providers/openstack/infra_manager.rb:30]: @kbrock does this depend on changes in core?  Or was this supposed to be `supports_not :shutdown` ?

Comment (kbrock) [reply app/models/manageiq/providers/openstack/infra_manager.rb:30]: it was a flub. thanks

**Comment by kbrock** [comment]
update:
- fixed `supports_not`
- fixed cops

**Review by agrare — APPROVED** [review]

**Pull request merged**